components/Banner/index.jsx

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head';
```

Comment:
This file is no needed. You can remove it.

Reply:
remove